code/go/0chain.net/blobbercore/datastore/postgres.go

```diff
@@ -59,10 +64,51 @@ func (store *postgresStore) GetTransaction(ctx context.Context) *gorm.DB {
 	if conn != nil {
 		return conn.(*gorm.DB)
 	}
-	Logger.Error("No connection in the context.")
+	logging.Logger.Error("No connection in the context.")
 	return nil
 }
 
 func (store *postgresStore) GetDB() *gorm.DB {
 	return store.db
 }
+
+func (store *postgresStore) AutoMigrate() error {
+
+	err := store.db.AutoMigrate(&Migration{})
+	if err != nil {
+		logging.Logger.Error("[db]", zap.Error(err))
+	}
+
+	latest := &Migration{}
+	result := store.db.Raw(`select * from "migrations" order by "version" desc limit 1`).First(latest)
```

Comment:
version is a string
let's say a comparison of "21.1.0" and "3.1.0" happens. it'll consider "3.1.0" bigger among them

Reply:
hmm. got it. let me convert it to int first.

Reply:
the above is not an issue because of versioning mechanism, but string sorting is different that integer sorting
which will fetch wrong latest version for you

Reply:
done

Reply:
get latest version by created_at instead of version